x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/sources_logic.ts

```diff
@@ -144,11 +144,17 @@ export const SourcesLogic = kea<MakeLogicType<ISourcesValues, ISourcesActions>>(
   selectors: ({ selectors }) => ({
     availableSources: [
       () => [selectors.sourceData],
-      (sourceData: SourceDataItem[]) => sourceData.filter(({ configured }) => !configured),
+      (sourceData: SourceDataItem[]) =>
+        sourceData
+          .filter(({ configured }) => !configured)
+          .sort((a, b) => a.name.localeCompare(b.name)),
```

Comment:
Seems like we could DRY this out a bit. We use the same logic in 4 places, once being in the other file. Do you think it would be beneficial to extract the sort logic into a `sortItemsByName` (or similar) util and share it in these 2 files?

Reply:
Upon reflection, I think you're right and this should probably be extracted. Also helped me remember that `.sort` sorts in place, which is generally not what we want.